includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -958,6 +958,9 @@ public function output_html() {
 	}
 	
 	public function preview_ubl() {
+		// get last settings
+		$this->settings = ! empty( $this->latest_settings ) ? $this->latest_settings : $this->get_settings( true );
```

Comment:
May I ask why we need to have this line?

Reply:
To display the document with the last settings.